src/Symfony/Bundle/WebServerBundle/WebServerBundle.php

```diff
@@ -13,6 +13,13 @@
 
 use Symfony\Component\HttpKernel\Bundle\Bundle;
 
+/**
+ * @deprecated since version 4.4, to be removed in 5.0; the new symfony local server has more feature, you should use it instead.
+ */
 class WebServerBundle extends Bundle
 {
+    public function boot()
+    {
+        @trigger_error('Using the WebserverBundle is deprecated since 4.4, the new symfony local server has more feature, you should use it instead.', E_USER_DEPRECATED);
```

Comment:
```suggestion
        @trigger_error('Using the WebserverBundle is deprecated since 4.4. The new symfony local server has more features, you should use it instead.', E_USER_DEPRECATED);
```